### PR TITLE
Update nginx to modern versions on ja docs

### PR DESCRIPTION
**Pull request by cakecatz**
~Example's nginx version and that description's nginx version are different.~

~This PR fixes the example's nginx version.~

Update nginx to modern versions on ja docs.
It also fixes wrong version of nginx.

https://kubernetes.io/ja/docs/concepts/workloads/controllers/deployment/#creating-a-deployment

<details>
<summary>Screenshots</summary>

image

image
</details>

### CODE REVIEW

File: content/zh/examples/controllers/nginx-deployment.yaml [hunk 1]
```diff
@@ -16,6 +16,6 @@ spec:
     spec:
       containers:
       - name: nginx
-        image: nginx:1.15.4
```
Comment (tanjunchen) [inline content/zh/examples/controllers/nginx-deployment.yaml]: see https://github.com/kubernetes/website/blob/master/content/en/examples/controllers/nginx-deployment.yaml#L19

thanks for your contribution.
could you sync this to /en right now?
/hold

Comment (tanjunchen) [reply content/zh/examples/controllers/nginx-deployment.yaml]: For different languages, try to submit PR separately, thank you

**Comment by inductor** [comment]
FYI https://github.com/kubernetes/website/pull/19198
@tanjunchen @cakecatz  @markthink

File: content/zh/examples/controllers/nginx-deployment.yaml [hunk 2]
```diff
@@ -16,6 +16,6 @@ spec:
...
+        image: nginx:1.7.9
```
Comment (tanjunchen) [inline content/zh/examples/controllers/nginx-deployment.yaml]: see https://github.com/kubernetes/website/pull/19198/files#diff-dc77252173c201b05594bf5a6a059eceR19

Comment (tanjunchen) [reply content/zh/examples/controllers/nginx-deployment.yaml]: nginx:1.15.4 -> nginx:1.7.9?

File: content/ja/examples/controllers/nginx-deployment.yaml [hunk 3]
```diff
@@ -16,6 +16,6 @@ spec:
     spec:
       containers:
       - name: nginx
-        image: nginx:1.15.4
+        image: nginx:1.7.9
```
Comment (inductor) [inline content/ja/examples/controllers/nginx-deployment.yaml]: It has to be changed too

File: content/zh/examples/controllers/nginx-deployment.yaml (hunk 1 above)
Comment (cakecatz) [reply content/zh/examples/controllers/nginx-deployment.yaml]: > could you sync this to /en right now?

OK!
I rebased commit to latest master.

> For different languages, try to submit PR separately

I dropped zh changes from this PR.

Thank you 😄

File: content/zh/examples/controllers/nginx-deployment.yaml (hunk 2 above)
Comment (cakecatz) [reply content/zh/examples/controllers/nginx-deployment.yaml]: I rebased so it was updated to latest version!

File: content/ja/examples/controllers/nginx-deployment.yaml (hunk 3 above)
Comment (cakecatz) [reply content/ja/examples/controllers/nginx-deployment.yaml]: I fetched upstream and rebased commits to that master.
So this lines updated to modern nginx version.

File: content/ja/docs/concepts/workloads/controllers/deployment.md [hunk 4]
```diff
@@ -28,7 +28,7 @@ Deploymentによって作成されたReplicaSetを管理しないでください
 
 下記の項目はDeploymentの典型的なユースケースです。
 
-* ReplicaSetをロールアウトするために[Deploymentの作成](#creating-a-deployment)を行う: ReplicaSetはバックグラウンドでPodを作成します。Podの作成が完了したかどうかは、ロールアウトのステータスを確認してください。
```
Comment (inductor) [inline content/ja/docs/concepts/workloads/controllers/deployment.md]: I dont think these changes are necessary. The blanks are set on purpose as it means the same as <br> tag in HTML.

File: content/ja/docs/concepts/workloads/controllers/deployment.md [hunk 5]
```diff
@@ -615,7 +615,7 @@ Deploymentのローリングアップデートは、同時に複数のバージ
 
 ユーザーは1つ以上の更新処理をトリガーする前に更新の一時停止と再開ができます。これにより、不必要なロールアウトを実行することなく一時停止と再開を行う間に複数の修正を反映できます。
 
-* 例えば、作成直後のDeploymentを考えます。  
+* 例えば、作成直後のDeploymentを考えます。
```
Comment (inductor) [inline content/ja/docs/concepts/workloads/controllers/deployment.md]: ditto

File: content/zh/examples/controllers/nginx-deployment.yaml (hunk 1 above)
Comment (tanjunchen) [reply content/zh/examples/controllers/nginx-deployment.yaml]: @cakecatz
Thanks
/hold cancel

File: content/ja/docs/concepts/workloads/controllers/deployment.md (hunk 4 above)
Comment (cakecatz) [reply content/ja/docs/concepts/workloads/controllers/deployment.md]: Ahh, my editor removed that whitespaces... 
I pushed commit to restore this.

**Comment by inductor** [comment]
Merging is blocked due to #19809, but it looks good to me.

**Comment by sftim** [comment]
/hold
pending v1.18 Kubernetes release

**Comment by VineethReddy02** [comment]
/hold cancel
v1.18 release is out.

**Comment by tanjunchen** [comment]
@VineethReddy02 Thanks
/lgtm

**Pull request merged**